README.rst

```diff
@@ -1,96 +1,90 @@
+This plugin is just like other django applications except for the parts that really make it able to be integrated with edx-platform and to understand those you might want to take a look at `this link. <https://github.com/edx/edx-django-utils/tree/master/edx_django_utils/plugins>`_
+
+Features
+~~~~~~~~
+
+edx-sysadmin provides different features such as:
+
+* Register Users:
+    * You can ``register new user accounts`` with an easy to use form via ``Users`` tab.
+* Delete Courses:
+    * You can ``delete any course by using a course ID or directory`` via ``Courses`` tab.
+* Git Import:
+    * You can ``import any course maintained through a git repository`` via ``Git Import`` tab.
+* Git Logs
+    * You can ``check the logs for all imported courses`` through git via ``Git Logs`` tab.
+
+Installing The Plugin
+~~~~~~~~~~~~~~~~~~~~~
+
+* You can install the plugin into your Open edX environment using PyPI e.g. ``pip install edx-sysadmin`` or directly from github e.g. ``pip install https://github.com/mitodl/edx-sysadmin.git``
+* Once you have installed the plugin you can visit ``<EDX_BASE_URL>/sysadmin`` to access the plugin features.
+* If you decide to make your own changes in the plugin you can go to ``Development Workflow`` section below.
 
-Documentation
--------------
+``Note``: In some cases you might need to restart edx-platform after installing the plugin to reflect the changes.
 
-(TODO: `Set up documentation <https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs>`_)
 
 Development Workflow
 --------------------
 
 One Time Setup
 ~~~~~~~~~~~~~~
+
 .. code-block::
 
   # Clone the repository
-  git clone git@github.com:edx/edx-sysadmin.git
+  git clone git@github.com:mitodl/edx-sysadmin.git
   cd edx-sysadmin
-
-  # Set up a virtualenv using virtualenvwrapper with the same name as the repo and activate it
-  mkvirtualenv -p python3.8 edx-sysadmin
-
+  # Set up a virtual environment and activate it
+  virtualenv -p python3.8 venv_sysadmin && source venv_sysadmin/bin/activate
 
 Every time you develop something in this repo
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. code-block::
 
   # Activate the virtualenv
-  workon edx-sysadmin
-
+  source venv_sysadmin/bin/activate
+  # Get into edx-sysadmin repo directory
+  cd edx-sysadmin
   # Grab the latest code
   git checkout master
   git pull
-
   # Install/update the dev requirements
   make requirements
-
-  # Run the tests and quality checks (to verify the status before you make any changes)
-  make validate
-
   # Make a new branch for your changes
   git checkout -b <your_github_username>/<short_description>
-
   # Using your favorite editor, edit the code to make your change.
-  vim …
-
-  # Run your new tests
-  pytest ./path/to/new/tests
-
-  # Run all the tests and quality checks
-  make validate
```

Comment:
How we run tests in plugin? I mean there should be an alternate of `make validate` command to run unit and quality tests.

Reply:
As Arslan has created a new PR (https://github.com/mitodl/edx-sysadmin/pull/19) in which he changed the way tests will be run, I am currently looking into it but seems like on local dev the tests will only be run after integrating with edx.

Reply:
pylint and pytest both are dependent on edx modules.

Reply:
We should then write some notes how we can install sys admin in development mode in edx-platform devstack and how we can run tests in lms-shell of devstack.

Reply:
I have added the integration testing details now.